src/lib/OpenEXR/ImfCompressor.cpp

```diff
@@ -162,6 +166,7 @@ numLinesInBuffer (Compression comp)
         case B44_COMPRESSION:
         case B44A_COMPRESSION:
         case DWAA_COMPRESSION: return 32;
+        case ZSTD_COMPRESSION: return 1;// FIXME
```

Comment:
Why does this needs fixing ?

Reply:
I am still unclear about single vs multi scanline compression. Will probably assume single for now.